auth-web/src/composables/affiliations-factory.ts

```diff
@@ -82,6 +82,18 @@ export const useAffiliations = () => {
   /** Returns the status of the affiliation. */
   const status = (business: Business): string => {
     if (isTemporaryBusiness(business)) {
+      // Reflect 'Pending | Payment Incomplete' status
+      if (business.status === 'PENDING') {
+        return BusinessState.PENDING
+      }
+      if (business.status) {
+        return BusinessState[business.status]
+      }
```

Comment:
I'm confused why you need lines 86-88. Won't like 90 handle it?

Reply:
Agree, I will fix it.

Reply:
Also, isn't there always a `status` property in the business object? In this case, the following code will not be run...?

Reply:
1.  When the business has a effectiveDate and no other draft statuses (like "AWAITING_REVIEW" and "CHANGE_REQUESTED"), the draftStatus fetched from api is PAID. So the business.status would be PAID. No need to check business.effectiveDate again.
2. When the business has a effectiveDate and has a draft status, the draftStatus (like "AWAITING_REVIEW") will override PAID. The business.status would be the draftStatus fetched from api, so PAID (Effective Effective) won't show.
3. Same for PENDING